roles/config-wireguard/config/ams.yml

```diff
@@ -243,3 +243,10 @@ wg_peers:
   wg_pubkey: 1JwYl6O6Y94PiKe5dXcgwMYjDxRK3im8mqvbZIeMSDE=
   peer_v4: null
   peer_v6: fe80::1525
+
+- name: dn42-hexxnet
+  port: 20225
+  remote: dn42peering.sarkoziadam.hu:40233
+  wg_pubkey: zajOi1myXS0VZeYdNKcK3W1ROIcGSQtQ1Ie17ZrNxiI=
+  peer_v4: 172.22.128.179
+  peer_v6: fe80::1080:0225
```

Comment:
Please pick a link-local address outside `fe80::1080:x` to avoid future conflicts. I suggest something like `fe80::225` (last digits of your asn) or `fe80::225:1080`